specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/workspaceRP.json

```diff
@@ -3181,6 +4633,41 @@
         }
       }
     },
+    "WorkspaceConnectionOAuth2": {
+      "description": "ClientId and ClientSecret are required. Other properties are optional\r\ndepending on each OAuth2 provider's implementation.",
+      "type": "object",
+      "properties": {
+        "authUrl": {
+          "description": "Required by Concur connection category",
+          "type": "string"
+        },
+        "clientId": {
```

Comment:
>clientId [](http://example.com/codeflow?start=9&length=8)

Please indicate format/type of ID in the name and/or description. If it's a supported format type (arm-id, url, uuid, etc.), add a format: as well.